src/reducers/targetReducer.js

```diff
@@ -0,0 +1,14 @@
+import { createReducer } from '@rootstrap/redux-tools';
+import { createTargetSuccess } from 'actions/targetActions';
+
+const initialState = {
+  targets: [],
+};
+
+const handleCreateTargetSuccess = (state, { payload }) => {
+  state.targets = state.targets.concat(payload);
```

Comment:
not sure if you want to concat, what's the response from this? only one target? or the whole list of targets

Reply:
I think the concat is fine because the response return only the target created. It's just one target. You can see here: https://targetmvd.docs.apiary.io/#reference/0/target-collection/create-target

Reply:
It returns only the new created target. Maybe you can `push` instead.

Reply:
I think push is not correct for this case. Because I need a new array to save in state.targets and 'push' only return the length of the new array and if I use push I would be affecting the immutability. 
Maybe another option more clear will be '[...state.targets, payload]'.